src/services/bcryptService.js

```diff
@@ -0,0 +1,20 @@
+const bcrypt = require('bcrypt');
+
+function createHash(password) {
+  return bcrypt.hash(password, 10);
```

Comment:
```suggestion
  return bcrypt.hash(password, salt);
```

it's better to save salt in .env file.